docs/reference/content/driver/reference/fle.md

```diff
@@ -0,0 +1,166 @@
++++
+date = "2019-06-13T09:00:01+01:00"
+title = "Field Level Encryption"
+[menu.main]
+  parent = "Sync Reference"
+  identifier = "Sync Field Level Encryption"
+  weight = 30
+  pre = "<i class='fa'></i>"
++++
+
+# Field Level Encryption
+
+New in MongoDB 4.2 field level encryption allows administrators and developers to encrypt specific data fields in addition to other 
+MongoDB encryption features.
+
+With field level encryption, developers can encrypt fields client side without any server-side 
+configuration or directives. Client-side field level encryption supports workloads where applications must guarantee that 
+unauthorized parties, including server administrators, cannot read the encrypted data.
+
+## Installation
+
+The recommended way to get started using field level encryption in your project is with a dependency management system. 
+Field level encryption requires additional packages to be installed as well as the driver itself.  
+See the [installation]({{< relref "driver/getting-started/installation.md" >}}) for instructions on how to install the MongoDB driver. 
+
+{{< distroPicker >}}
+
+### libmongocrypt
+
+There is a separate jar file containing`libmongocrypt` bindings.
+
+{{< install artifactId="mongodb-mongocrypt" version="1.0.0-SNAPSHOT">}}
+
+If the jar fails to run there are separate jar files for specific architectures:
+
+#### RHEL 7.0*
+{{< install artifactId="mongodb-mongocrypt" version="1.0.0-SNAPSHOT" classifier="linux64-rhel70">}}
+
+#### OSX*
+{{< install artifactId="mongodb-mongocrypt" version="1.0.0-SNAPSHOT" classifier="osx">}}
+
+#### Windows*
+{{< install artifactId="mongodb-mongocrypt" version="1.0.0-SNAPSHOT" classifier="win64">}}
+
+#### Ubuntu 16.04
+{{< install artifactId="mongodb-mongocrypt" version="1.0.0-SNAPSHOT" classifier="linux64-ubuntu1604">}}
+
+
+* Distribution is included in the main `mongodb-mongocrypt` jar file.
+
+### mongocryptd configuration
+
+`libmongocrypt` requires the `mongocryptd` daemon / process to be running. A specific daemon / process uri can be configured in the 
+`AutoEncryptionSettings` class by setting `mongocryptdURI` in the `extraOptions`.
+
+{{% note class="important" %}}
+If not configured the driver will automatically try to start the daemon / process in: `/tmp/mongocryptd.sock`. This requires
+the `jnr.unixsocket` library to be installed.
+{{% /note %}}
+
+More information about libmongocrypt will soon be available from the official documentation.
+
+
+### Examples
+
+The following is a sample app that assumes key and schema have already been created in MongoDB and the `encryptedField` field is encrypted:
```

Comment:
Maybe mention that for the purpose of the example, we're using a local key but that AWS KMS is also an option.

Reply:
Done